pkg/controller/backup_deletion_controller_test.go

```diff
@@ -739,6 +741,265 @@ func TestBackupDeletionControllerProcessRequest(t *testing.T) {
 		// Make sure snapshot was deleted
 		assert.Equal(t, 0, td.volumeSnapshotter.SnapshotsTaken.Len())
 	})
+
+	t.Run("backup is not downloaded when there are no DeleteItemAction plugins", func(t *testing.T) {
```

Comment:
I've used the same test set up as the previous tests but there is a lot of repetition here now and I think they could be refactored. I'm happy to do that in this PR or I can do a follow up PR.

Reply:
I think you should open a ticket (to track) and do this separately.